docs/customer-360/customer-journey/process.sql

```diff
@@ -0,0 +1,24 @@
+SET 'auto.offset.reset' = 'earliest';
+
+-- Create stream of pages
+CREATE STREAM pages (
+  customer INTEGER,
+  time BIGINT,
+  page_id STRING,
+  page STRING
+) WITH (
+  VALUE_FORMAT = 'JSON',
+  KAFKA_TOPIC = 'pages',
+  PARTITIONS = 6
+);
+
+-- Create stateful table with Array of pages visited by each customer, using the `COLLECT_LIST` function
+-- Get `COUNT_DISTINCT` page IDs
```

Comment:
Not sure if important, but should we point out that `COUNT_DISTINCT` is not exact, but probabilistic?

Reply:
@mjsax for users who want to read more about this function, https://github.com/confluentinc/ksqldb-recipes/pull/131/commits/c84823500d4b320f63994322df59ab656dd4947d adds a link to documentation.